chapters/dashboard/self-host.md

```diff
@@ -0,0 +1,21 @@
+## Self-host a Dashboard
+
+One can self-host a dashboard locally.
+
+1. `git clone https://github.com/jina-ai/dashboard.git && cd dashboard`.
+2. Install dependencies using command `yarn`.
+3. Run dashboard via the following ways .
+
+### Run in the debug mode
+
+1. `node testServer`
```

Comment:
We have other scripts for this now. 

`npm run dev`

runs both dev server and react app in parallel.

So, I guess we have to make a call on telling users the easiest way, explain different parts of it or both.